zscaler_api_talkers/zia/talker.py

```diff
@@ -285,6 +285,67 @@ def list_admin_users(
 
         return self._obtain_all(url)
 
+    def add_adminUsers(self, loginName, userName, email, password, role, comments='',
```

Comment:
PR looks good. Please add variable type and return type. You can mimic other calls. thanks